ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx

```diff
@@ -123,17 +123,14 @@ function PolicyBehaviorForm({ hasActiveViolations }: PolicyBehaviorFormProps) {
 
     // Node event sources do not support scoping, exclusions, or enforcement actions
     function onChangeNodeEventSource() {
-        setValues(
-            {
-                ...values,
-                eventSource: 'NODE_EVENT',
-                scope: [],
-                excludedImageNames: [],
-                excludedDeploymentScopes: [],
-                enforcementActions: [],
-            },
-            false
```

Comment:
Aha - this prevents a round of validation which in turn prevents the user from selecting "Next" if the only action is to change the lifecycle to Runtime->Node.